test/models/user_test.rb

```diff
@@ -0,0 +1,57 @@
+require "test_helper"
+
+describe User do
+  describe 'relations' do
+    it 'can set the user id through new vote' do
+      # Create two models
+      @user = User.create!(username: 'puppy')
+      @vote = Vote.new
```

Comment:
I think it's good/okay that you made two models here (you definitely need a user and a vote), but don't forget that you can also reference the fixtures that are already made user and vote.